pkgs/by-name/mo/motif/package.nix

```diff
@@ -90,6 +99,11 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  # provide correct configure answers for cross builds
+  configureFlags = [
+    "ac_cv_func_setpgrp_void=yes"
```

Comment:
This would be wrong for BSD, right?

Reply:
Thank you so much!

Reply:
Done :)

Makes me think we should have a catch-all list of guaranteed-correct configure flags... would speed ./configure up, and would have helped out during the GCC 14 update too. (Lots of old configure scripts use too minimal test programs, which tripped warnings-made-errors in GCC 14.)

Reply:
Yeah that would actually be really nice to have in stdenv.  I think I've seen other distros do similar (Void Linux?).